src/modules/display.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable */
+import { removeTask } from './addAndRemove.js';
+import { editItem, saveItem } from './EditAndSave.js';
+import { complete } from './checkAndClear.js';
+
+const activityToDo = document.getElementById('activity-to-do');
+
+export const display = () => {
+  const lists = localStorage.getItem('toDoList');
+  const get = JSON.parse(lists);
+  let activity = [];
+  if (get) {
+    activityToDo.innerHTML = '';
+    activity = get;
+    activity.forEach((item, indexNo) => {
+      item.index = indexNo+1;
+      const list = document.createElement('li');
+      // list.id = indexNo;
```

Comment:
Please try to avoid pushing comments with the code. You can always return to the previous version using `git history`.